rgb-contract-implementation/schema/supported-schemas.md

```diff
@@ -0,0 +1,132 @@
+# List of supported Schemas
+
+This section lists the schemas that have been officially implemented so far, describing the data
+they work with and the list of supported operations.
+
+## NIA (Non Inflatable Asset)
+
+This schema defines a non-inflatable fungible asset, involving the following data:
+- Ticker: short identifier of the asset, to be displayed on wallets and exchanges (e.g.: BTC, USDT, ...). Note
+    that there is no guarantee of uniqueness
+- Name: extended name of the asset (e.g.: Bitcoin, Tether, ...)
+- Details: optional string describing the asset in more detail
+- Precision: Number of decimal places of the standard representation of the asset (e.g.
+    would be 8 for BTC and 2 for USD)
+- Contract Terms: contains a string describing the contract and an optional media
+    attachment
+- Issued Supply: total amount of tokens created at issuance. Since the asset is not
+    inflatable, it also serves as an upper bound on the circulating supply
+
+_Note:_ Ticker, Name, Details and Precision are represented by a single object called _AssetSpec_
+
+The following operations are supported for the asset:
+- Transfer: send some amount of assets to a number of destinations (optionally including
+    change); it supports multiple inputs in case multiple allocations should be merged in
+    order to build up the desired output amount.
+
+## UDA (Unique Digital Asset)
+
+This schema defines a unique non-fungible asset (NFT), involving the following data:
+- Ticker: short identifier of the asset, to be displayed on wallets and exchanges (e.g.: BTC, USDT, ...). Note
+    that there is no guarantee of uniqueness
+- Name: extended name of the asset (e.g.: Bitcoin, Tether, ...)
```

Comment:
BTC and USDT are not good examples of UDA assets. No good ones come to mind, I'd just drop the examples

Reply:
it might make sense to only keep the main definition (i.e. "Ticker: short identifier of the asset") in each asset's section and group the longer field explanations in a dedicated section (e.g. a glossary or note section, with "Ticker: to be displayed on wallets and exchanges (e.g.: BTC, USDT, ...)")